cli/azd/pkg/ext/models.go

```diff
@@ -70,6 +70,9 @@ type HookConfig struct {
 	Windows *HookConfig `yaml:"windows,omitempty"`
 	// When running on linux/macos use this override config
 	Posix *HookConfig `yaml:"posix,omitempty"`
+	// Environment variables in this list are added to the hook script and if the value is a akvs:// reference
+	// it will be resolved to the secret value
+	Secrets map[string]string `yaml:"secrets,omitempty"`
```

Comment:
Just curious, do you feel that it is common for 

```yaml
secrets:
  - KV_SECRET:KV_SECRET
```

to be used; and if so, does:

```yaml
secrets:
  - KV_SECRET
```

feel better as syntactic sugar?